src/CommunityToolkit.Maui.Core/Essentials/Badge/Android/BadgeFactory.android.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Android.Content;
+using Android.Content.PM;
+
+namespace CommunityToolkit.Maui.ApplicationModel;
+
+/// <summary>
+/// Factory for <see cref="IBadge"/>
+/// </summary>
+public static class BadgeFactory
+{
+	static readonly Dictionary<string, IBadgeProvider> providers = new();
+	static readonly DefaultBadgeProvider defaultBadgeProvider = new();
+
+	/// <summary>
+	/// Register provider for launcher type
+	/// </summary>
+	/// <param name="launcherType">Launcher type</param>
+	/// <param name="provider">Provider implementation</param>
+	public static void SetProvider(string launcherType, IBadgeProvider provider)
```

Comment:
Let's change this to `SetBadgeProvider` to match the other public method name, `GetBadgeProvider`

```suggestion
	public static void SetBadgeProvider(string launcherType, IBadgeProvider provider)
```